sdk/core/Azure.Core/tests/samples/OperationSamples.cs

```diff
@@ -51,6 +52,28 @@ public async Task OperationUpdateStatus()
             #endregion
         }
 
+        [Test]
+        [Ignore("Only verifying that the sample builds")]
+        public async Task OperationCustomPollingCompletion()
+        {
+            // create a client
+            SecretClient client = new SecretClient(new Uri("http://example.com"), new DefaultAzureCredential());
+
+            #region Snippet:OperationCustomPollingCompletion
+            // Start the operation
+            DeleteSecretOperation operation = await client.StartDeleteSecretAsync("SecretName");
+
+            while (operation.HasCompleted)
+            {
+                // there could be more complex logic to control polling interval
+                Thread.Sleep(TimeSpan.FromSeconds(60));
```

Comment:
If the example is going to be async, we can do this instead.

```suggestion
                await Task.Delay(TimeSpan.FromSeconds(60));
```